Makefile

```diff
@@ -58,6 +59,16 @@ test-go: ## Run tests
 manager: lint-full ## Build manager binary
 	go build -o bin/manager sigs.k8s.io/cluster-api/cmd/manager
 
+.PHONY: docker-build-manager
+docker-build-manager:
+	docker run --rm \
+	-v "${PWD}:/go/src/github.com/kubernetes-sigs/cluster-api" \
+	-v "${GOPATH}/bin:/go/bin" \
```

Comment:
Since we're using go modules, we probably shouldn't assume `GOPATH` is set. I would probably make this `${PWD}/bin/manager`.

Reply:
All addressed.  Added `lint-full` in line 63
Please review again. Thanks!